src/modules/market/components/market-trading--form/market-trading--form.jsx

```diff
@@ -18,89 +18,196 @@ class MarketTradingForm extends Component {
     orderType: PropTypes.string.isRequired,
     orderPrice: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
     orderQuantity: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
+    marketQuantity: PropTypes.string.isRequired,
     orderEstimate: PropTypes.string.isRequired,
     selectedOutcome: PropTypes.object.isRequired,
     nextPage: PropTypes.func.isRequired,
     updateState: PropTypes.func.isRequired,
     isMobile: PropTypes.bool.isRequired,
     minPrice: PropTypes.number.isRequired,
     maxPrice: PropTypes.number.isRequired,
-    availableFunds: PropTypes.instanceOf(BigNumber).isRequied
+    availableFunds: PropTypes.instanceOf(BigNumber).isRequired
   }
 
   constructor(props) {
     super(props)
 
     this.INPUT_TYPES = {
```

Comment:
Would it be more natural to define these constants in `modules/markets/constants/market-types`, instead of attaching them to the component?